docs/admin/daemons.md

```diff
@@ -92,6 +92,12 @@ when the pod is created, so it is ignored by the scheduler).  Therefore:
  - DaemonSet controller can make pods even when the scheduler has not been started, which can help cluster
    bootstrap.
 
+## How Daemon Pods work with taints and tolerations
+
+Daemon pods will not be deleted by `NodeController` in case of node problems, such as when node turns to
+`Unreachable` or `NotReady`. The reason is that the `NodeController` added an infinite tolerations of
```

Comment:
This sentence doesn't read well to me and it is quite long. Consider replacing "the reason" and rephrasing "added an infinite tolerations". 

Toleration is a verb and should not include an "s".